docs/csharp/index.yml

```
@@ -101,17 +157,13 @@ landingContent:
           - text: Version compatibility
             url: whats-new/version-update-considerations.md
 
-  - title: "C# language reference"
+  - title: "Stay in Touch"
```

I really like this addition. (Same nit):

```suggestion
  - title: "Stay in touch"
```